draft-ietf-quic-version-negotiation.md

```diff
@@ -212,17 +231,36 @@ Note also that the client can disable compatible version negotiation by
 only including the Chosen Version in the Other Versions field of the Version
 Information Transport Parameter.
 
-If the server does not find a compatible version, it will use the original
-version if it supports it, and if it doesn't then the server will perform
-incompatible version negotiation instead, see {{incompat-vn}}.
+If the server does not find a compatible version (including the
+client's chosen version), it will perform incompatible version negotiation instead,
+see {{incompat-vn}}.
+
+Note that it is possible to have incompatible version negotation
+followed by compatible version negotiation. For instance, if A and B
+are compatible and versions C and D are compatible, the following
+scenario may occur:
+
+~~~
+    Client                                          Server
+
+    Chosen = A, Other Versions = (A, B) ----------------->
+    <------------------------ Version Negotiation = (D, C)
+
+    Chosen = C, Other Versions = (C, D) ----------------->
+    <-------------------------------------- Negotiated = D
```

Comment:
Want to show Other Versions here too?

Fun fact: that list could include B and E.

Reply:
I feel like adding this could add more confusion than it'd help remove?

Reply:
Oh yes, for certain, but you might at least show that there are other versions here, so maybe "Other Versions = (...)".

Reply:
Good point, wrote up PR #105